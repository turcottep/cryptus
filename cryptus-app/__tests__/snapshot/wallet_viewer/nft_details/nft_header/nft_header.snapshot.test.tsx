import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import NftHeader from "../../../../../components/profile/wallet_viewer/nft_details/nft_header/nft_header";

describe("<NftHeader />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<NftHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
