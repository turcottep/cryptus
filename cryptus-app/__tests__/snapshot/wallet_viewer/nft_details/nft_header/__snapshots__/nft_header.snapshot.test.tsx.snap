// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NftHeader /> should display properly 1`] = `
<div>
  <a
    onClick={[Function]}
    target="_blank"
  >
    <img
      src="/icons/back_icon.png"
    />
  </a>
  <div
    onClick={[Function]}
  >
    <img
      src="/icons/more_icon.png"
    />
  </div>
</div>
`;
